tag_enum.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+r"""Script for generating an IntEnum of DICOM tags from pydicom's dictionary"""
+from argparse import ArgumentParser, Namespace
+from pathlib import Path
+from typing import Final
+
+from pydicom.datadict import DicomDictionary
+
+
+HEADER: Final = r"""#!/usr/bin/env python
```

Comment:
Could we store this definition in an actual Python file, and then read that file in as text which we append to (or append to a copy of that template file)? That way we could have type/syntax checking for the code.

Reply:
I would be open to modifying the current script so that `_tag_enum.py` contains the header / methods, and the dictionary derived tags in this file are updated when the script is run. I'd rather not have three separate files (script, hearder file, and output file) if we can avoid it.